plugin/pkg/admission/limitranger/admission.go

```diff
@@ -106,15 +105,6 @@ func (l *LimitRanger) runLimitFunc(a admission.Attributes, limitFn func(limitRan
 		return nil
 	}
 
-	obj := a.GetObject()
-	name := "Unknown"
-	if obj != nil {
-		name, _ = meta.NewAccessor().Name(obj)
-		if len(name) == 0 {
-			name, _ = meta.NewAccessor().GenerateName(obj)
```

Comment:
why do we do name generating here?

Reply:
good question. Who wrote this?

Reply:
/shrug

we should never try generating name outside the store registry, amiright?